ckanext/validation/fanstatic/js/module-resource-schema.js

```diff
@@ -98,18 +127,41 @@ this.ckan.module('resource-schema', function($) {
         .on('click', this._onRemoveURL)
         .insertBefore(this.field_url_input);
 
+      // TODO: disable?
```

Comment:
@amercader 
Can you please take a look how it works now? 

TBH I feel like I'm missing something regarding your comment here - https://github.com/frictionlessdata/ckanext-validation/issues/34#issuecomment-380232871. 

Of course, I'm pretty neutral on enabling/disabling the linked schema editing - just curious why the user can't edit the linked schema based on the same approach we have for the create resource page.